adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -999,6 +999,17 @@ object ADAMContext {
   // Add ADAM Spark context methods
   implicit def sparkContextToADAMContext(sc: SparkContext): ADAMContext = new ADAMContext(sc)
 
+  /**
+   * Creates an ADAMContext form SparkSession. Sets active session, including SQLContext, to input session.
+   *
+   * @param ss SparkSession
+   * @return ADAMContext
+   */
+  def ADAMContextFromSession(ss: SparkSession): ADAMContext = {
+    SparkSession.setActiveSession(ss)
```

Comment:
OOC, why do you need the `setActiveSession` call?

Reply:
This resets the sparkContext to read the session passed in. This fixes the issue if one sparkContext has already been started in the scala backend, then it will be replaced.